ledger-core/v2/README.md

```diff
@@ -37,41 +52,81 @@ To test Insolar locally, install it and deploy as described below.
+   
+### Test
 
-   * Requester:
+#### Benchmark test
 
-     ```
-     bin/apirequester -k=.artifacts/launchnet/configs/ -p=http://127.0.0.1:19101/api/rpc
-     ```
+When the pulse watcher says `INSOLAR STATE: READY`, you can run a benchmark in another terminal tab/window:
 
-     The requester runs a scenario: creates a number of users with wallets and transfers some money between them. For the first time, the script does it sequentially, upon subsequent runs — concurrently.
+   ```
+    bin/benchmark -c=4 -r=25 -k=.artifacts/launchnet/configs/
+   ```
 
      Options:
-     * `-k`: Path to the root user's key pair. All requests for a new user creation must be signed by the root one.
-     * `-p`: Node's public API URL. By default, the first node listens on the `127.0.0.1:19101` port. It can be changed in configuration.
+     * `-k`: Path to the root user's key pair.
+     * `-c`: Number of concurrent threads in which requests are sent.
+     * `-r`: Number of transfer requests to be sent in each thread.
+
+#### Functional tests
+
+Functional tests run via a requester that runs a complex scenario: creates a number of users with wallets and transfers some money between them.<br>
+Upon the first run, the script does it consecutively, upon subsequent runs — concurrently.
 
-   * Benchmark:
+When the pulse watcher says `INSOLAR STATE: READY`, run the following command in another terminal tab/window:
 
-     ```
-     bin/benchmark -c=4 -r=25 -k=.artifacts/launchnet/configs/
-     ```
+   ```
+    bin/apirequester -k=.artifacts/launchnet/configs/ -p=http://127.0.0.1:19101/api/rpc
+   ```
 
      Options:
      * `-k`: Path to the root user's key pair.
-     * `-c`: Number of concurrent threads in which requests are sent.
-     * `-r`: Number of transfer requests to be sent in each thread.
+             All requests for a new user creation must be signed by the root one.
+     * `-p`: Node's public API URL. By default, the first node listens on the `127.0.0.1:19101` port. 
+             It can be changed in configuration.
+
+
+## Run single Insolar node
+
+We use a two-process approach for running a node in production.<br>
+"Two-process" means we insulate the network layer into a separate process and give a higher priority to time-critical consensus and network algorithms:
+
+* The first insolard process contains the network layer and runs with a higher priority and resource quota (via the `nice` command or Linux namespaces). You can identify this process by the `passive` flag.
+
+* The second insolard process is responsible for other node components.
+
+You have two options to run Insolar node in production:
+
+ * First process manages the second.
+ 
+ * A watchdog manages each process separately.
+ 
+In case of the first option, you can use a systemd service script:
+
+```
+insolard config generate --role=virtual -c ./node.conf
```

Comment:
I tried to run it on the branch and get an error
aalekseev@MacBook-Pro-4 v2 % ./bin/insolard config generate --role=virtual -c ./node.conf 
Error: unknown flag: --role

Reply:
well, I forgot to rebuild bin -_-
but after that, I also get an error

./bin/insolard config generate --role=virtual -c ./node.conf
{"level":"info","writeDuration":"3.000µs","message":"TODO: generate configuration with config=./node.conf role=virtual mode="}
{"level":"fatal","stack":"main.configGenerateCommand.func1\n\t/Users/aalekseev/go/src/github.com/insolar/assured-ledger/ledger-core/v2/cmd/insolard/commands.go:169","writeDuration":"0.000µs","message":"not implemented"}

is it ok?

Reply:
Yes, it's ok, coz this PR is only about CLI commands. Most of them still not implemented